index.js

```diff
@@ -22,7 +21,7 @@ if (!global.L) {
     var L = require(leafletPath);
     global.L = L;
 
-    var scriptLength = leafletPath.split(path.sep).slice(-1)[0].length;
+    var scriptLength = leafletPath.split('/').slice(-1)[0].length;
```

Comment:
ups, I forgot it, I've updated now but tests are not working good, I'm trying to fix them but node-canvas is working nicely, it fixes my problem